src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -255,30 +259,44 @@ public bool AddTimeData(IPAddress peerAddress, TimeSpan offsetSample, bool isInb
         /// <remarks>
         /// The caller of this method is responsible for holding <see cref="lockObject"/>.
         /// <para>
-        /// The function takes a single copy of each inbound sample and combines them with <see cref="OutboundToInboundWeightRatio"/>
+        /// The function takes a single copy of each inbound sample and combines them with <see cref="OffsetWeightSecurityConstant"/>
         /// copies of outbound samples.
         /// </para>
         /// <para>
-        /// We require to have at least <see cref="MinUnweigtedSamplesCount"/> unweighted samples to change the value of <see cref="timeOffset"/>.
+        /// When there are many more inbound samples than unweighted outbound, which could be the case
+        /// in a malicious attack, the security is still maintained by using a dynamic inbound/outbound multiplier ratio on the outbound samples 
+        /// that maintains the accepted level of security. 
+        /// For example, an <see cref="OffsetWeightSecurityConstant"/> of 3 gives us 33.3% protection.
+        /// An <see cref="OffsetWeightSecurityConstant"/> of 2 gives us 25% protection.
```

Comment:
This discussion on the level of protection with different values belongs rather to the comment of the constant itself, not here really